# CONTROLLERS USED IN THE EXAMLE
joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100
# THIS CONFIGURATION IS ACTUALLY SENDING TAU = K*Q
joint_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints:
        - joint_1
        - joint_2
        - joint_3
        - joint_4
        - joint_5
        - joint_6
